src/index.html

```diff
@@ -3,17 +3,15 @@
 <html>
 
   <head>
-    <link rel="stylesheet" type="text/css" href="scss/application.css">
+    <link rel='stylesheet' type='text/css' href='scss/application.css'>
+    <link rel='stylesheet' type='text/css' href='scss/reset.css'>
```

Comment:
the reset should be added before the application because you want the application to override the reset when the selectors have the same specificity